server/server.js

```diff
@@ -228,13 +228,11 @@ const intercomClient = !isTrue(process.env.DISABLE_INTERCOM) ? new IntercomOffic
 const useTranslateApi = isTrue(process.env.SHOULD_USE_TRANSLATION_API);
 let translateClient = null;
 if (useTranslateApi) {
-  const GOOGLE_CREDS_TEMP_FILENAME = ".google_creds_temp";
-
-  fs.writeFileSync(GOOGLE_CREDS_TEMP_FILENAME, process.env.GOOGLE_CREDS_STRINGIFIED);
-
-  translateClient = Translate({
-    projectId: JSON.parse(fs.readFileSync(GOOGLE_CREDS_TEMP_FILENAME)).project_id,
-  });
+  // Tell translation library where to find credentials, and write them to disk.
+  process.env.GOOGLE_APPLICATION_CREDENTIALS = '.google_creds_temp'
```

Comment:
This was undocumented before, but I think it was required to tell the Google client libs where to find the credentials via the `GOOGLE_APPLICATION_CREDENTIALS` envvar, in order for it to load them (since `.google_creds_temp` is a nonstandard place to look).

Reply:
cc @metasoarous as to whether this is the right path going forward — is disk required here?

Reply:
to clarify, disk was always required :)

We were writing this file before:
https://github.com/pol-is/polisServer/pull/319/files#diff-ea75b530d74f94670cea053feb8b961bL231-L233

but we weren't hardcoding this envvar, that tells _the Google client library_ where to find the credentials we'd written to disk -- presumably we were setting `GOOGLE_APPLICATION_CREDENTIALS` as a heroku envvar (e.g., `GOOGLE_APPLICATION_CREDENTIALS=.google_creds_temp`) or else the google library would've had no way to pick up the credentials that were written to this filepath by the old server code.